pasteit/public/javascripts/pasted.js

```diff
@@ -16,4 +17,7 @@ $(document).ready(function() {
 
     editor.getDoc().setValue($("#pasted-content").val());
     editor.refresh();
+
+    $("#delete").prop("href", "/delete"+window.location.pathname);
```

Comment:
Where's the linter man? spaces between + and the strings.

Reply:
done in 25e11d9